backend/__main__.py

```diff
@@ -0,0 +1,4 @@
+import app
+
+if __name__ == "__main__":
+    app.create_app().run()
```

Comment:
```suggestion
    webapp = app.create_app()
    webapp.run()
```
Just so that we can maybe make config changes or whatever to the `webapp` object later.